src/draw/DefaultDrawer.js

```diff
@@ -193,66 +210,41 @@ class DefaultDrawer {
   }
 
   /**
-   * Initialize all draw option of the components.
+   * Initializes all draw option of the components.
+   * @param {Object} d3Element - D3 selected elements.
    * @param {Component[]} components - List of Components.
-   * @param {Object} d3Element - D3 selected element.
+   * @param {String} parentId - ID of parent it contains components.
```

Comment:
`ID of parent it contains components.` -> `ID of parent which contains components.`

Reply:
Done